content/app/github/wrappers/line-element-factory.js

```diff
@@ -0,0 +1,92 @@
+define("content/app/github/wrappers/line-element-wrapper", function(require, exports, module) {
+
+	// imports
+	var LineElementWrapper = require("content/app/github/wrappers/line-element-wrapper");
+	var ExpandableLineElementWrapper = require("content/app/github/wrappers/expandable-line-element-wrapper");
+
+	/**
+	 * Wrapper for line DOM element.
+	 */
+	var LineElementFactory = Class(Object, {
+
+		$statics : {
+
+			TAG_NAME : 'tr',
+
+			/**
+			 * Populate expandable line ranges.
+			 * @param  {AbstractLineElementWrapper[]} lines
+			 */
+			_populateExpandableRanges : function (lines) {
+
+				for (var i = 0; i < lines.length; i++) {
+					var line = lines[i];
+
+					if (line instanceof ExpandableLineElementWrapper) {
+						var prevOldLineNumber = 0;
+						var nextOldLineNumber = Number.MAX_VALUE;
+						var prevNewLineNumber = 0;
+						var nextNewLineNumber = Number.MAX_VALUE;
+
+						var prevLine = lines[i-1];
+						if (prevLine) {
+							prevOldLineNumber = prevLine.oldNumber;
+							prevNewLineNumber = prevLine.newNumber;
+						}
+
+						var nextLine = lines[i+1];
+						if (nextLine) {
+							nextOldLineNumber = nextLine.oldNumber;
+							nextNewLineNumber = nextLine.newNumber;
+						}
+
+						line.oldNumberRange = [prevOldLineNumber+1, nextOldLineNumber-1];
+						line.newNumberRange = [prevNewLineNumber+1, nextNewLineNumber-1];
```

Comment:
Good point, I've made an issue (#15) for that case and will add comment in code. I'd like to merge this into master as soon as possible.